content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -153,6 +153,34 @@ section refers to several key workload enablers and how they map to Windows.
   * `emptyDir` volumes
   * Named pipe host mounts
   * Resource limits
+  * OS field: `pod.spec.os.name` should be set to `windows` to indicate that the current pod is a Windows pod. 
+              Please note that there is no effect of having value for this field since it is behind a featuegate `IdentifyPodOS` and it is currently alpha.
+              {{< note >}}
+              If the `pod.spec.os.name` field is set to `windows`, following fields must be unset on the Windows pods when the `IdentifyPodOS` featuregate is enabled:
```

Comment:
Are there fields that need to be unset when the Podfield is set to `linux`?  Such as the `securityContext.WindowsOptions`?  Are we documenting that anywhere?

Reply:
This should be available in the API docs for the field. Since we have a separate documentation for the Windows pods, I am mentioning it explicitly.

Reply:
Thanks, I remember those docs now.  Makes sense to mention it here but I do wonder about maintaining this list in to places.

Reply:
I am not sure which other place you're recommending.

Reply:
This looks like the same list here: https://github.com/kubernetes/kubernetes/blob/c1153d3353bd4f4b68d85245d53d2745586be474/pkg/apis/core/types.go#L2968-L2990 which will be published as API docs here: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/ once the does are published.

My main concern here is these two will get out of sync.

Reply:
So, you don't want this list to be mentioned here in our Windows docs?

Reply:
I think it should be mentioned but there should be one place where all the fields are listed otherwise it will get out sync and be confusing.